<?xml version="1.0" ?>
<sdf version="1.6">
  <world name="default">
    <gravity>-2 0 -9.8</gravity>

    <include>
      <uri>model://ground_plane</uri>
    </include>
    <include>
      <uri>model://sun</uri>
    </include>

    <include>
      <uri>model://trisphere_cycle</uri>
      <name>trisphere_cycle_slip0</name>
      <pose>0 0 0  0 0 0</pose>
      <plugin name="wheel_slip_front" filename="libgazebo_ros_wheel_slip.so">

        <ros>
          <namespace>trisphere_cycle_slip0</namespace>
          <parameter name="slip_compliance_unitless_lateral" type="double">0.3</parameter>
          <parameter name="slip_compliance_unitless_longitudinal" type="double">0.4</parameter>
        </ros>

        <wheel link_name="wheel_front">
          <wheel_normal_force>77</wheel_normal_force>
        </wheel>
      </plugin>
    </include>

  </world>
</sdf>
